clash-vexriscv/src/ffi/impl.cpp

```diff
@@ -51,11 +53,23 @@ VVexRiscv* vexr_init()
 {
 	contextp = new VerilatedContext;
 	VVexRiscv *v = new VVexRiscv(contextp);
-	Verilated::traceEverOn(true);
 	v->clk = false;
 	return v;
 }
 
+VerilatedVcdC* vexr_init_vcd(VVexRiscv *top, const char* path)
+{
+	if (path != NULL){
```

Comment:
Only a style nitpick, since we don't have a formatter for the C++ code set up: for if/while/for/... statements we have a space between the parentheses and the opening brace.
The other new `if`s in the file are the same. 

```suggestion
	if (path != NULL) {
```